frontend/src/components/form/tiptap/TiptapEditor.vue

```diff
@@ -259,13 +266,32 @@ export default {
       })
     },
   },
+  mounted() {
+    document.removeEventListener('keydown', this.specialKeyListeners)
+    document.removeEventListener('keyup', this.specialKeyListeners)
+    document.removeEventListener('contextmenu', this.specialMenuListeners)
```

Comment:
What is the reason that these are not in the 'unmounted' lifecycle?

Reply:
Because unmounted is a vue3 lifecycle 😉

Reply:
Oops.
But is it usual for a `removeEventListener` to be implemented before `addEventlistener`? Doesn't seem particularly symmetrical / particularly trustworthy.